docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -1784,3 +1784,30 @@ To enable cross origin requests to the remote api add the flag
 "--api-enable-cors" when running docker in daemon mode.
 
     $ docker -d -H="192.168.1.9:2375" --api-enable-cors
+
+
+## 3.4 Errors
+
+If there is an error while processing a request, the daemon will generate
```

Comment:
Yep, putting JSON in the header will allow for good reuse - and should mean we only one one header instead of multiple.